docs/howto/how-to-grant-api-access.md

```diff
@@ -94,8 +94,10 @@ rules:
   verbs:
   - '*'
 ```
-**Note**: The `'*'` in verbs allows all actions on the specified resources
-In each case, replace `<your-api-group>` and `<your-custom-resources>` with the actual API group and resource names provided by the installed cluster extension.
+
+!!! note
+    The `'*'` in verbs allows all actions on the specified resources
```

Comment:
Does this need a period (`.`) at the end?

Reply:
Yes. Good catch! Updated.